xstate-chatbot/nodejs/src/machine/chat-machine.js

```diff
@@ -21,9 +22,39 @@ const chatStateMachine = Machine({
         context.slots = {};
       }),
       on: {
-        USER_MESSAGE: 'selectLanguage',
+        USER_MESSAGE: 'mvOfficer',
       },
     },
+    mvOfficer: {
+      id: 'mvOfficer',
+      initial: 'process',
+      states: {
+        process: {
```

Comment:
Directly "process"! No prompt state??

Reply:
Actually this chatbot will remain same for covid like after missed call then normal flow of covid flows 
But a keyword 'PBGIS' will be shared with gis nodal officer and after entering this keyword only ,GIS flow starts.